prow/integ-galley-local-tests.h

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Should most of this file be in separate common file and reused here and another files?

Reply:
Created a common set of files and called specific targets from there (ala e2e shell file).